conf/messages.en

```diff
@@ -297,8 +297,10 @@ supplementary.eori = EORI number
 supplementary.eori.info = Enter the EORI number or business details
 supplementary.eori.hint = The number starts with a country code, for example FR for France, and is then followed by up to 15 digits
 
+supplementary.eori.nongb.error=EORI number must start with GB followed by 12 to 15 digits
```

Comment:
I like it that you distinguish between two cases, but `nongb` is not a good prefix in my opinion. This message will be also presented for `GB1234567890ABC`.
I'd suggest name like `supplementary.eori.error.format` and the other error to have `.length` suffix.

Reply:
Merged to one message after discussing with James Watson